.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,51 @@
+name: Test and Build
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build-python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --no-cache poetry>=1.0.2 twine
+          poetry install
+      - name: Check codestyle
+        run: |
+          poetry run black --check notebooks
+      - name: Test
+        env:
+          COGNITE_CLIENT_SECRET: ${{ secrets.COGNITE_CLIENT_SECRET }}
```

Comment:
how do you set up the secrets?

Reply:
In the Security setting, Secret section

Reply:
I don't see it. I might not have access to it. Should every member of the geospatial team be added as an "owner" of the repo, maybe?

Reply:
yeap, I'll update some permission/access.

Reply:
added.

Reply:
@vincent-cognite Can you check if access is available?

Reply:
yes, I can update or delete secrets under "settings -> secrets" 👍